Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -2528,7 +2528,7 @@ def splunk_search_command(service: client.Service, args: dict) -> CommandResults
 def splunk_job_create_command(service: client.Service, args: dict):
     query = args['query']
     app = args.get('app', '')
-    if not query.startswith('search'):
+    if not query.startswith('search') and not query.startswith('|'):
```

Comment:
It seems there’s already a function called build_search_query that performs this operation. You might be able to use it instead of creating new logic.